app/views/admin/sponsors/new.html.haml

```diff
@@ -17,6 +17,11 @@
                 = f.input :website, as: :url
             .row
               .large-3.columns
+                = f.input :contact_first_name
+              .large-3.columns
+                = f.input :contact_surname
+            .row
+              .large-3.columns
```

Comment:
This should also include an email address. (Our sponsors dont need to be codebar members)